src/app/legacy/containers/ArticleMediaPlayer/index.jsx

```diff
@@ -24,14 +24,15 @@ const pageTypeMap = {
 
 const ArticleMediaPlayerContainer = ({ blocks }) => {
   const { id, pageType } = useContext(RequestContext);
+  const hasPlaceholder = pageType !== 'mediaArticle';
```

Comment:
Could maybe use the `MEDIA_ARTICLE_PAGE` const here

Reply:
Yeah good spot, I might as well replace pageTypeMap keys as well while I'm here

Reply:
Ah yea, good shout 👍